docs/source/pythonprimer.rst

```diff
@@ -101,7 +104,37 @@ To pass the function to another function (as in callbacks in Python Script), jus
 Line 11 calls the ``callFunc`` function, passing the printNameAge function to it.  The ``callFunc`` function then calls printNameAge on line 8.
 
 
+Scope
+-----
 

```

Comment:
There is also a possibility to create scripts that are used by other scripts by importing them. 
In such a case, a module scope has now been introduced.

Reply:
Do you mean that with module scope was not possible so far (with older versions) in PS? 

I would still choose to only talk about local and global scopes there : as you can tell, this python primer was meant to be appealing even to non-python programmers, and give just the minimum needed to be able to create working scripts. 

Additionally, I'm not familiar with module scope... So if you insist, you can add that yourself later.

Reply:
No this has been possible since the start of PS.
If you create a script and run it directly, it is exactly as you described,
but if you create a.py script

```py
x = 5
```

and a second script b.y, which is then executed

```py
import a
# x is not globally known, instead it must be used as
print(a.x) # here is now the scope on module level
print(x) # throws an exception
```

But I agree, it might already go too deep for a primer.

Reply:
Thanks for the explanation. I get that it's simply a script-level (file-level) scope, for all items defined in a.py, available via `a.name` in b.py .